### PR TITLE
ci(suite-desktop): gpg sign artefacts in codesign targets

### PR DESCRIPTION
**Pull request by prusnak**
Adds artefact signing with GPG.

This needs to be tested in the `codesign` branch

### CODE REVIEW

File: packages/suite-desktop/scripts/gnupg-sign.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+if [ -z "$GPG_CSC_USERID" -o -z "$GPG_CSC_LINK" -o -z "$GPG_CSC_KEY_PASSWORD" ]; then
+  echo "GPG sign skipped"
+  exit 0
+fi
+
+gpg --batch --import "$GPG_CSC_LINK"
+
+for f in packages/suite-desktop/build-electron/*.{AppImage,dmg,exe} ; do
```
Comment (tsusanka) [inline packages/suite-desktop/scripts/gnupg-sign.sh]: Does it make sense to sign also dmg and exe?

**Review by tsusanka — APPROVED** [review]
LGTM, did not test

File: packages/suite-desktop/scripts/gnupg-sign.sh (hunk 1 above)
Comment (prusnak) [reply packages/suite-desktop/scripts/gnupg-sign.sh]: Yes, we do it already and the signatures are linked from the landing page. It certainly does not hurt.

Comment (tsusanka) [reply packages/suite-desktop/scripts/gnupg-sign.sh]: Ok, true :)

**Review by keraf — APPROVED** [review]

**Comment by prusnak** [comment]
Squashed and merged to develop as 53a7aa7a073ed677088323a7163c10afeb65f19c

**Pull request closed without merging**